src/main/python/model/postbuilder.py

```diff
@@ -134,18 +152,126 @@ def post_type_changed(self, index):
         self.seasonField.setVisible(is_hidden)
         self.seasonLabel.setVisible(is_hidden)
 
+    def load_tmdb_info(self):
+        tmdbID = self.movidDBIDField.text().strip()
+        if tmdbID == '':
+            self.__search_tmdb()
+        else:
+            self.__get_tmdb_details(tmdbID)
+
+    def __search_tmdb(self):
+        url = 'https://api.themoviedb.org/3/search/movie'
+
+        if self.postTypePicker.currentIndex() == 1:
+            url = 'https://api.themoviedb.org/3/search/tv'
+            params = {
+                "api_key": "5e23b4412adb55e7cca19cfb9d0196b6",
+                "query": self.titleField.text().strip(),
+                "first_air_date_year": self.yearField.text().strip(),
+                "include_adult": 'false'
+            }
+        else:
+            params = {
+                "api_key": "5e23b4412adb55e7cca19cfb9d0196b6",
```

Comment:
I assume this is your api key? I'd think this should be stored via preferences instead (unless they allow sharing api keys?). Not sure about their T&Cs for this sort of use.

Reply:
I notice their v4 api users a Bearer token btw, not sure if they plan to remove v3 but perhaps worth bearing in mind

Reply:
Sharing keys are a little bit of a gray area.  Since they're free and no restrictions it's really not a big deal. Someone from TMDB said that in a forum post.  So I wasn't worried about it with how light our user base and use is for it and figured it was easier for the couple guys that are using it.  I can switch it over to a pref though if you're not comfortable with it. 

The v4 api with bearer looks mainly to be for user auth and not app auth.  We're not loading favorites, lists, etc. and only doing app auth to query movie data.